src/server/server.cpp

```diff
@@ -1,16 +1,15 @@
-//
-// Created by zanna on 04/10/18.
-//
 #include "server.h"
 
-server::Server::Server(uint16_t port) : port_(port){}
+server::Server::Server(uint16_t port) : port_(port) {}
 
 void server::Server::stop() {
     stopped_ = true;
 }
 
 void server::Server::signal_handler(int signal_number) {
     /* TODO: Put exit cleanup code here. */
```

Comment:
Do we still need to clean up resources before exiting?

Reply:
Probably

Reply:
Are we going to fix this issue in this branch?

Reply:
no

Reply:
ok